### ISSUE TITLE
Configure geolocation_service to run on DOCKER

### ISSUE THREAD

**Issue by brunolcarli** [sosbrumadinho/brumadinho_location#94]
# Português-BR

Seguinte, pretendo colocar o serviço pra rodar em um servidor (a princípio heroku) mas gostaria que rodasse em um container, porém não tenho muita experiência com isso. Estou no trabalho agora e se alguém puder configurar pra que ele rode num container com docker ele vai rodar em qualquer ambiente e ja agilizaria o deploy para o heroku.

Se alguém puder agilizar essa parte eu agradeço, do contrário começarei a pesquisar e fazer o trabalho hoje (01/02/19) a partir das 22:00.

# English

Ok, i pretend to put the service to run on a host server (thinking on Heroku) and i like it to run on a docker container, buti don have much experience with this. Im at work right now and if someone could do  this configs to the service to run on docker container it will run anywhere and speed up the deploy processo to heroku server.

If anyone could speed up this task i'll appreciate it,  otherwise i will start the research today (01/02/19) at 10:00pm.

**Comment by pictos** [comment]
Estamos correndo atrás de patrocínio em relação a isso. Olhando com empresas que possuem serviços em nuvens e vendo se elas podem nos oferecer hospedagem de graça. Acho uma boa ideia esperar um pouco mais antes de começar a implantar isso.

**Comment by dezoito** [comment]
@brunolcarli, ia fazer uma composição de containeres usando Django e Nginx para servir o API, mas vi pela msg do @pictos para aguardar.

Me avisem se houver uma definição e se ainda quiser ajuda para montar a parte em docker, ok?

**Comment by brunolcarli** [comment]
@dezoito sim, estou esperando avaliação do meu último PR, a mais de 10 dias. Pelo visto a galera não tem muita pressa então...

Talvez a gente consiga uma release antes de limparem toda a lama, reflorestarem e repopularem toda região xD

**Comment by pictos** [comment]
@dezoito o apoio de empresas pode demorar um pouco por conta da burocracia. Infelizmente esse é um processo lento. E sim precisaremos de ajuda na implantação do Docker.

@brunolcarli , peço desculpas pela demora. Mas temos poucos devs para fazer o code review. No momento a maioria está trabalhando em outros PRs e demandas mais criticas.
